auth/naive.go

```diff
@@ -65,3 +65,6 @@ func (*NaiveUserManager) GetOrCreateUser(u gimlet.User) (gimlet.User, error) {
 func (b *NaiveUserManager) ClearUser(u gimlet.User, all bool) error {
 	return errors.New("Naive Authentication does not support Clear User")
 }
+func (*NaiveUserManager) GetGroupsForUser(string) ([]string, error) {
+	return []string{}, nil
```

Comment:
should we return an error?

Reply:
I think it's a bit moot at the moment - eventually this should return something meaningful enough so that trying to run a dummy instance with naive auth will allow access to features